modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/custom/facet/display/context/builder/CustomFacetDisplayContextBuilder.java

```diff
@@ -204,9 +204,7 @@ private BucketDisplayContext _buildBucketDisplayContext(
 		bucketDisplayContext.setBucketText(term);
 		bucketDisplayContext.setFilterValue(term);
 
-		int frequency = termCollector.getFrequency();
-
-		bucketDisplayContext.setFrequency(frequency);
+		bucketDisplayContext.setFrequency(termCollector.getFrequency());
 
 		bucketDisplayContext.setFrequencyVisible(_frequenciesVisible);
 
```

Comment:
@mbowerman i think we can also delete the empty lines 208 and 210?

Reply:
ninja'd...

Reply:
I thought so too but I didn't do it since (1) the purpose of this pull was just to fix the failing SF, and (2) There was already an empty line between setFrequencyVisible and setSelected which didn't seem necessary to me either, so I didn't want to mess with empty lines that I didn't understand.